netvisor/services/accounting.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+    netvisor.services.accounting
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    :copyright: (c) 2013-2016 by Fast Monkeys Oy.
+    :license: MIT, see LICENSE for more details.
+"""
+from inflection import camelize
+
+from ..requests.accounting import AccountingListRequest
+from .base import Service
+
+
+def format_dates(date):
+    try:
+        return date.strftime('%Y-%m-%d')
+    except AttributeError:
+        return date
+
+
+class AccountingService(Service):
+    def list(self, **kwargs):
+        query = {
+            camelize(key): format_dates(value)
```

Comment:
I would prefer a more explicit approach. You cannot see what query parameters there are without going to Netvisor's documention.